src/v0/destinations/am/transform.js

```diff
@@ -131,8 +131,10 @@ function createRevenuePayload(message, rawPayload) {
 
 function updateTraitsObject(property, traitsObject, actionKey) {
   const propertyToUpdate = getValueFromMessage(traitsObject, property);
-  traitsObject[actionKey][property] = propertyToUpdate;
-  deleteObjectProperty(traitsObject, property);
+  if (property && property.length > 0) {
```

Comment:
Need to first check that `property` is a string.

Reply:
Done